README.md

```diff
@@ -5,57 +5,70 @@ SPDX-License-Identifier: Apache-2.0
 
 # Welcome to the DAML repository!
 
-This repository hosts all code for the DAML smart contract language, originally created by [Digital Asset](https://www.digitalasset.com).
+This repository hosts all code for the DAML smart contract language, originally created by
+[Digital Asset](https://www.digitalasset.com).
 
 ## To start using DAML
 
-See our documentation at [docs.daml.com](https://docs.daml.com).
+To download DAML, follow [the installation instructions on docs.daml.com](https://docs.daml.com/getting-started/installation.html).
+To try out using it, follow the [quickstart guide](https://docs.daml.com/getting-started/quickstart.html).
 
-Download the DAML SDK Developer Preview at [https://www.daml.com](https://www.daml.com).
-
-If you have questions about how to use DAML or how to build DAML-based solutions, please ask them on [StackOverflow using the `daml` tag](https://stackoverflow.com/tags/daml).
+If you have questions about how to use DAML or how to build DAML-based solutions, please ask
+them on [StackOverflow using the `daml` tag](https://stackoverflow.com/tags/daml).
 
 ## To start contributing to DAML
 
 We warmly welcome [contributions](./CONTRIBUTING.md).
 
-### Cloning this repository
+### 1. Clone this repository
 `git clone git@github.com:DACH-NY/daml.git`.
 
-### Setting up the DA Development Environment (also known as `dev-env`)
-dev-env is used to provide dependencies required during build phase, like java,
-bazel, and python for some tooling scripts, while the code itself is built through
-bazel.
+### 2. Set up the DA Development Environment ("`dev-env`")
+
+`dev-env` provides dependencies required during build phase, like Java, Bazel, and Python
+for some tooling scripts. The code itself is built using Bazel.
+
+#### Set up `dev-env` on Linux or macOS
 
-#### Linux and macOS
 1. Install Nix: `bash <(curl https://nixos.org/nix/install)`
-2. Enter dev-env: `eval "$(dev-env/bin/dade assist)"`
+2. Enter `dev-env`: `eval "$(dev-env/bin/dade assist)"`
+
+If you don't want to `dev-env` manually each time using `eval "$(dev-env/bin/dade assist)"`,
```

Comment:
Is "to dev-env" a verb or is there an "enter" missing?

Reply:
Good spot, fixed!

Reply:
👍